packages/mcontrolcenter-bin/mcontrolcenter-bin.pacscript

```diff
@@ -0,0 +1,31 @@
+name="mcontrolcenter-bin"
+gives="mcontrolcenter"
+version="0.3.2"
+depends="libqt5widgets5"
+url="https://github.com/dmitry-s93/MControlCenter/releases/download/${version}/MControlCenter-${version}.tar.gz"
+repology=("project: mcontrolcenter")
+description="An application that allows you to change the settings of MSI laptops."
+hash="bab563f692fb3d42494d95de21611d230e0b0736e2e80f08cf8a1b7b990d9f0f"
+maintainer="সৌম্যদীপ <soumyadeepghosh2004@zohomail.in>"
+
+install() {
+	cd app
+	sudo install -Dm755 ${gives} "$pkgdir"/usr/bin/${gives}
+	sudo install -Dm644 ${gives}.desktop $pkgdir/usr/share/applications/${gives}.desktop
+	sudo install -Dm644 ${gives}.svg "$pkgdir"/usr/share/icons/hicolor/scalable/apps/${gives}.svg
+	sudo install -Dm755 ${gives}-helper "$pkgdir"/usr/libexec/${gives}-helper
+	sudo install -Dm644 ${gives}-helper.conf "$pkgdir"/usr/share/dbus-1/system.d/${gives}-helper.conf
+	sudo install -Dm644 ${gives}.helper.service "$pkgdir"/usr/share/dbus-1/system-services/${gives}.helper.service
+}
+postinst() {
+	if ask "Do you want this programme to start at startup?" Y; then
+  sudo -u ${PACSTALL_USER} mkdir ${homedir}/.config/autostart
```

Comment:
```suggestion
  sudo -u ${PACSTALL_USER} mkdir -p "${homedir}/.config/autostart/"
```

Reply:
A .config folder is anways gonna exist

Reply:
Maybe not autostart. It's better to just make sure it exists rather than not.